blob/s3blob/s3blob.go

```diff
@@ -199,10 +199,16 @@ func (o *URLOpener) OpenBucketURL(ctx context.Context, u *url.URL) (*blob.Bucket
 		if err != nil {
```

Comment:
Do the tests pass? I would expect V2ConfigFromURLParams to fail if you passed it a URL with these new parameters in it.

Reply:
I have added the following tests to catch this issue:
https://github.com/google/go-cloud/pull/3491/files#diff-0520a4d136ddd4348403e11bf2af92e43dd4dae87d9d970370c795cb966614f6R311

Similarly, in NewConfig:
https://github.com/google/go-cloud/pull/3491/files#diff-0520a4d136ddd4348403e11bf2af92e43dd4dae87d9d970370c795cb966614f6R222

Now if someone add a new query param in any of the place, this test will be able to catch this issue.